recipes/nbexamples/meta.yaml

```diff
@@ -9,7 +9,7 @@ package:
 source:
   fn: {{ gh_repo }}-v{{ version }}.tar.gz
   url: https://github.com/{{ gh_org }}/{{ gh_repo }}/archive/v{{ version }}.tar.gz
-  sha256: 96926c36333e57aee3820722f1107fbf6a3df9113b8585b14e572fccce385923
+  sha256: f391b42ea6856d92a06fafa18ede464e400c9b2804a2a022d0b7197fd4bd4061
```

Comment:
Well, if you think about it, this issue could be happening with other GitHub repos/tags. So it is better we start getting to the bottom of it right away.

That being said, this was pretty easy to do. 😄